### ISSUE TITLE
[Other] No VA.gov Experience Standard for the issue found. (00.00.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54709]
#### VFS team name
GovCIO-VFEP

#### VFS product name
GI Bill Comparison Tool

#### VFS feature name
Institution Ratings System

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
[This was encountered while reviewing the institution ratings, but has been confirmed to be an existing problem in production.]

When arriving on a school profile page and allowing a screen reader to announce all content starting at the top of the page, after announcing the H1 (name of the school) the screen reader next announces the items under "Your estimated benefits," eg. "GI Bill pays to school..., Tuition and fees charged..." etc.

This skips over a lot of other content, and is incredibly confusing for a screen reader user who attempts to navigate through the page.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Activate a screen reader of your choice. (Recommend NVDA for Windows, VoiceOver for Mac.)
2. Navigate to a school profile, eg. [University of Maryland College Park](https://staging.va.gov/education/gi-bill-comparison-tool/institution/11800020).
3. Allow the screen reader to start reading the page.

**Expected behavior:** Once the page loads, content is read in DOM order starting from the H1 and proceeding down the page.

**Actual behavior:** Once the page loads, the H1 is read. Then, the screen reader skips straight to "Your estimated benefits" -- but does not read the "Your estimated benefits" heading. Instead, it begins reading from "GI Bill pays to school" and proceeds from there.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
I believe this is happening because the content in "Your estimated benefits" is wrapped in a div with some unexpected ARIA attributes:

```
<div aria-atomic="true" aria-live="polite" role="status">
```

I don't believe any of those attributes are necessary, and are likely what's causing this behavior. `role="status"` and `aria-live="polite" are intended to provide status updates to a user, eg. something has changed and that information needs to be communicated immediately before other content is announced. That doesn't fit this situation and leads to this disruption.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
